frontend/src/index.html

```diff
@@ -1,17 +1,21 @@
 <!DOCTYPE html>
-<html>
-    <head>
-        <meta charset="utf-8" />
-        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-        <title>Elba Package Repository</title>
-        <meta name="viewport" content="width=device-width, initial-scale=1" />
-        <link
-            href="https://fonts.googleapis.com/css?family=Fira+Mono|Source+Sans+Pro:400,600,700"
-            rel="stylesheet"
-        />
-    </head>
-    <body>
-        <div id="main-app"></div>
-        <script src="./index.tsx"></script>
-    </body>
-</html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>elba - Package manager for Idris</title>
+    <meta name="viewport"
+        content="width=device-width, initial-scale=1, shrink-to-fit=no ,minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
+    <meta name="theme-color" content="#834bb4" />
+    <link href="https://cdn.bootcss.com/font-awesome/5.7.2/css/all.min.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Fira+Mono|Poppins:400,500,600,700|Source+Sans+Pro"
```

Comment:
Could we do without Poppins or Source Sans Pro and cut down on the number of font weights? Right now the fonts exceed the size of the JS payload and CSS combined and I get the feeling one of the font families may be unnecessary. Your call on this however, since you own the design.